lib/Resizable.js

```diff
@@ -119,7 +119,10 @@ export default class Resizable extends React.Component<Props, ResizableState> {
             deltaY += deltaTopSinceLast / this.props.transformScale;
           }
         }
-        this.lastHandleRect = handleRect;
```

Comment:
Not that it matters much, but why not just save the whole rect?

Reply:
I hit an issue with flow upon trying to cast an exact object to an inexact type. I'm new to flow so couldnt find a proper way around this, just got it working by taking the exact data I needed. I'm open to changes though